### ISSUE TITLE
Complete source code

### ISSUE THREAD

**Issue by ghost** [AmedeeBulle/lp-water-level#1]
Hello,
First, thank you for the great project. I would like to ask you if you plan to share complete source code include Lorawan communication and so on? 

Thank you

**Comment by steve-heslouin** [comment]
Yes please sharing the real source code would be nice as this is pretty much the concept of hackster.io :)

**Comment by pioga75** [comment]
A great project. Could you please put the complete code (including LORAWAN support) ??
